src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -27,31 +27,33 @@ class WorkDetail extends React.Component {
    * @param {object} work
    * @return {string|null}
    */
-  workDetailsObject(work) {
```

Comment:
The docstring should be updated for this if you're removing the parameter. In general, I prefer to pass in parameters instead of getting the `this.prop....` value because then you can test the function in isolation if you wanted to. Doing it this way you can still test the function, but you have to mount/shallow render it and make sure to pass props.

Reply:
I changed it to the other format, but that was giving a eslint error so I changed that also now